src/app/ApplicationTemplate.Shared.Views/Shell.xaml

```diff
@@ -31,12 +28,13 @@
 		<local:DiagnosticsOverlay DataContext="{Binding DiagnosticsOverlay}"
 								  Visibility="{Binding IsDiagnosticsOverlayEnabled, Converter={StaticResource TrueToVisible}, FallbackValue=Collapsed}"
 								  Grid.RowSpan="2" />
+		-->
 
 		<splash:ExtendedSplashScreen x:Name="AppExtendedSplashScreen"
+									 WindowsDataTemplate="{StaticResource WindowsExtendedSplashScreenDataTemplate}"
```

Comment:
won't it make more sense to defined it in the style instead since it's also declared there

Reply:
Yes, I can do that, cleaner that way.